### ISSUE TITLE
Can't enter room onto Move Avatar action

### ISSUE THREAD

**Issue by Godatplay** [sarahgould/mosi#24]
I'm trying to set up a behavior where I leave one room by walking off the screen to the right and then have the leftmost cell in the next room have a Move Avatar action that moves me to an offset. But I can't enter that new room if that action is in the cell.

**Comment by sarahgould** [comment]
Hmm... I wasn't able to reproduce this. I created a new sprite, added a "Move Avatar" action on it, and put it on the left edge of a room. Then I moved into that room from the right, and immediately hit that sprite and triggered the action - moving me into the center of the room, in this case. Did I miss a step? Do you maybe have that sprite set as a wall?

**Comment by Godatplay** [comment]
Huh...me neither! I'm not sure what happened. At first I thought the issue had to do with stacking the sprite with the Move action on top of a "background" one, but I could reproduce it without that even. When I came upon this issue, at first my background sprite was a wall, but I removed it and replaced it with a non-wall version of the same thing. And when that didn't work, I just removed the background altogether and only used the Move action sprite. Maybe the issue was more related to the wall-based background not getting reset fully, and now it has been? Honestly I'm not sure what happened, but things are fine now.
